src/lib/notificationHandlerWebsocket.js

```diff
@@ -0,0 +1,52 @@
+'use strict'
```

Comment:
Including `Websocket` in the the filename seems unnecessary. We don't include other types of notifications anymore, right?